src/GADM.jl

```diff
@@ -14,6 +14,8 @@ code examples are "IND", "USA", "BRA".
 """
 isvalidcode(str) = match(r"\b[A-Z]{3}\b", str) !== nothing
 
+prefixgadm(country) = "GADM_$country"
+
```

Comment:
Please remove  this function and simply create a variable inside the download function `ID = "GADM_$country"`. We don't need function for everything :)